backend/src/v2/driver/driver.go

```diff
@@ -448,19 +449,28 @@ func initPodSpecPatch(
 	accelerator := container.GetResources().GetAccelerator()
 	if accelerator != nil {
 		if accelerator.GetType() != "" && accelerator.GetCount() > 0 {
-			q, err := k8sres.ParseQuantity(fmt.Sprintf("%v", accelerator.GetCount()))
-			if err != nil {
-				return nil, fmt.Errorf("failed to init podSpecPatch: %w", err)
+			acceleratorType, err1 := resolvePodSpecRuntimeParameter(accelerator.GetType(), executorInput)
+			if err1 != nil {
+				return nil, err1
```

Comment:
Not sure, but maybe you need this:
```suggestion
				return nil, fmt.Errorf("failed to init podSpecPatch: %w", err1)
```

Reply:
it's moved below, this err1 will just trickle the error from `resolvePodSpecRuntimeParameter` which should be sufficient for this case